src/dswx_sar/dswx_runconfig.py

```diff
@@ -181,6 +183,7 @@ def custom_sort(pol):
 
     return co_pol_list, cross_pol_list, pol_list
 
+
 def validate_group_dict(group_cfg: dict, workflow_name) -> None:
```

Comment:
Please check if the parameter `workflow_name` is used in this function.

Reply:
Yes. I'll delete `workflow_name` here. It was designed for `dswx_nisar`, but does not need to be included.